### ISSUE TITLE
[Request] Every mod in Abyss.

### ISSUE THREAD

**Issue by Blade3575** [cursey/kanan#121]
May as well cut to the chase. This request should save everyone else time and effort. Make sure you check back every hour or less to the Abyss thread for new features that we've implemented, definitely can't wait more than one hour before trying to get a new mod from Abyss implemented into kanan.

**Comment by ghost** [comment]
Salty af.

**Comment by Renharu** [comment]
...

**Comment by TheProffessor** [comment]
Actually, it doesn't appear to be salt in this case, it seems to be lighthearted joking. In all honesty I do feel it's actually pretty hilarious how the requests do come rather quickly. If anything, it shows that people enjoy both Kanan and Blade's work, so I'd say that's a positive thing.

**Comment by ghost** [comment]
> lighthearted joking

Sure.

**Comment by ghost** [comment]
@QewQew
Proffessor is correct, it is Blade joking and being sarcastic at the same time.  I know because I've been talking to him and the Proffessor talks to him as well.  I'm very transparent on most things, and try real hard to act with honor in mind.  I was going to stay silent, but at this point here's how this works.  I've changed my perceptions a lot and have seen middle and common ground as to what everyone is doing.  I'd challenge anyone else including yourself to be able to do the same.  I feel this is only fair.  Feeding the cycle of hate, ignorance, and conflict only detracts from everything we're all involved in.  You choose to do that, as well as choose to interpret anything and everything how you wish.  Please bear in mind this is choices that you have control over.  You cannot control anyone else, as I've stated to the others in this convo and many more.  How you conduct yourself is entirely up to you.  I'm not saying I'm perfect, but I do try very hard.  So there it is.  

Perhaps it may have been best it not be posted at all as an issue, but it was.  The drawbacks of text is that you cannot see emotion, see facial expressions--for that matter you really have no idea what a person's intent is.  So we address the actions.  Actions are what matter, and please do note I'm choosing my words carefully so as not to make this longer than necessary.  My suggestion, as I've told Blade and others.  Let the matter drop and stop fueling the fire.

@All Concerned
People will ask for features and that's a given.  The right moral and ethical thing to do is to approach the other person and ask.  Blade has a right to feel the way he does as does Step29--it's not for anyone to decide otherwise.  On the other hand, its understandable to a point what is trying to be done here.   There's nothing wrong with collaboration.  You all have many opportunities here and so much more that limiting yourselves closes off.  I'm well aware of past history and have done extensive research and digging.  I know the actual facts, but they're too numerous to even begin.  You should be focusing on what matters and none of this petty crap.  It is what it is.  Grow up already.  

-knocks heads together-

Please close this issue.

**Comment by ghost** [comment]
> Proffessor is correct, it is Blade joking and being sarcastic at the same time.

The way the post was worded clearly did not intend to be a joke. Sarcastic, obviously, but with no good intentions.

> Blade has a right to feel the way he does as does Step29.

They do, and no one says they aren't allowed to. But if Blade is going to act childish on these feelings, then there's going to be some backlash.

> Grow up already.

Tell that to Blade; this was never an issue until Blade brought it up, every time. He's clearly the one who needs to move on.

**Comment by Blade3575** [comment]
Figured since this was closed that the fighting would stop since everything that needed to be said, was. I guess this was not the case.

> The way the post was worded clearly did not intend to be a joke. Sarcastic, obviously, but with no good intentions.

Actually, it _was_ partly a joke, and part sarcasm. I was definitely annoyed, but also found it kind of funny just how quickly a unique patch that was implemented into Abyss was requested. Please take it as such. Also, as much as I hate to say it, kanan recreating patches from Abyss that were originally created by either Step29 or myself, is a problem that still exists; not something that magically disappeared as you suggested. That is all I have to say for this matter and I hope that this serve as a reminder that not everything may be as it seems. I'm sorry that you've been offended by my post.

**Comment by ghost** [comment]
> Also, as much as I hate to say it, kanan recreating patches from Abyss that were originally created by either Step29 or myself, is a problem that still exists; not something that magically disappeared as you suggested.

The problem I was addressing was your childlike tantrums, not the recreation of the patches, which is obviously ongoing.

**Comment by Blade3575** [comment]
> The problem I was addressing was your childlike tantrums, not the recreation of the patches, which is obviously ongoing.

Since I verified that it was a joke, and explained why, I am going to hope that you were just quoting your previous statement when saying "childlike tantrums" and were not still trying to attack my character. I will take it as such and leave the matter be, as I hope everyone else does with this now.

**Comment by ghost** [comment]
@QewQew and everyone else that feels they can actually apply this:

You are correct on the first part, but you did evade my point that both Proffessor and I talk to Blade all the time.  So we know exactly what is said and for the most part the intentions, how he feels, etc.  I can honestly say no one really actually knows each other as people.  This is where I have the issue with everyone involved.  I've helped everyone I cross if I can, and will continue to do so.  I don't pick sides and try to stay as neutral as possible.  However, sometimes I have to come to the forefront to say things that not everyone may realize or think about.  I admit I was rather aggressive in trying to stamp things out on the forums, because we really should be focusing on what we say we love doing--otherwise, why do it?  
I understand there's going to be backlash, but just because one can do something, doesn't necessarily mean that we should.  Exercise in restraint and self control by all parties involved would be a very welcome change.  Reading into situations doesn't help at all.  There are opportunities to resolve this and put this in the past.  It's really who's going to take the higher road that's going to win this--if there's any "winning" to be done.  This "war", and crap really needs to stop.  It doesn't reflect well on anyone.  All I said applies to everyone not just you or Blade, but I admit I was addressing you as it was you that "reacted".  (you are the only one that has control over your choices)

Finger pointing, and trying to draw attention to another's flaws only takes the focus off one's self and it  only serves one purpose--distraction.  We only have control over ourselves.  Everyone needs to move on.  You wouldn't believe how much time I "waste" trying to keep the peace and communicate between people in order to help them get what they need, to help them in seeing where another person is coming from.  This only works with those that have open minds though.  All it takes to solve this is for people to actually start talking to each other--not at each other and doing more listening than talking.  A little empathy goes a long way.

TL:DR version:
Everyone chill FFS.  Have a beer, a Mt Dew, or something. ;D

You and anyone else is welcome to join the Discord for mabimods that I made, or you can join the IRC.  I invited @cursey and @Poshwosh there with the purpose of fostering good will and communication.  PM me on the forums if you're interested or Draconis #8309 if you wish.  You'll find I try to be very open minded.

**Comment by Poshwosh** [comment]
It doesn't look like a joke in my eyes. Someone who has all the resources of Abyss, requesting them to be recreated where else he could have offered the resources, if anything, it's blunt trolling.

To add to that, you are misusing the issue ticket system, some people here spend their time finding problems and figuring out what's the cause, and that's what you use it for, to troll, and advertise? 
I mean, I didn't even know you before all of this happen, nor do I know you now, but I just expected you to be more professional regarding competition. They say "Change, stay, or leave the situation.", and there's nothing wrong with you being annoyed of this situation, but whining about it isn't good for anyone here, it's not going to make good.

Let me just say it plain out here, you can't copyright memory addresses of Mabinogi, they belong to Nexon, nor can you do so with simple "patch ideas" either, they are not patents and they belong to no one. 
People request and will continue to do so because naturally humans will pick what they identify as better. They'll take what they find best, and they'll improve on it any way they can while striving for perfection. It's just how consumers work, nobody can change that.
To add to that, most of the "Abyss modifications" here existed long before Abyss came to be. You discovered your own way to achieve the result just as much as cursey did, and many others more will in generations of Mabinogi to come, so in the end... it's all about competitiveness. Which is weird because Abyss has so many users. Are you that bored?
cursey credits you as a sign of respect, as well as with every other person who contributed to the Mabinogi modification scene that had their share of ideas here. She didn't even need to, it's called respect.

**Comment by cursey** [comment]
Hi,

@Blade3575 Please keep your passive aggressive grievances off and away from the issue system on this project. This is not the place for it and you've done it before.

@TheProffessor I don't agree with you. This is not just a "lighthearted joke" and I don't know why you would defend this behavior.

To everyone else, I think it goes without saying but I will merge _any_ pull requests that I think are good additions to kanan. I will also try fulfilling _any_ requests that I think are good additions to kanan. I will do both of these **regardless** of who did it first as long as they comply with our agreement of not reversing Abyss. I'm more than happy to give credits to the authors and original creators. If I miss any just make an issue and let me know (I've already said this in the `README`).
